metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -282,7 +282,8 @@ class ScalametaLanguageServer(
     } catch {
       case NonFatal(e) =>
         onError(e)
-        ShutdownResult(-1)
+        // FIXME: server shouldn't send shutdown response if there was no shutdown request
+        ShutdownResult()
```

Comment:
@olafurpg I think something wrong is happening here. Server shouldn't send a shutdown response on completion request failure.

Reply:
lol, no idea how that go there. feel free to remove the whole try/catch, errors get caught and logged on request now.

Reply:
Done in f3d9342 (+ same in `documentFormattingRequest`)